src/murmur/Messages.cpp

```diff
@@ -863,12 +863,6 @@ void Server::msgChannelState(ServerUser *uSource, MumbleProto::ChannelState &msg
 			return;
 		}
 
-		QRegExp re2("\\w");
-		if (re2.indexIn(qsName) == -1) {
```

Comment:
I'm in favor of at least checking if its length is > 0.

Would also be interesting to check if these same rules are applied for RPC.